addons/cookoff/functions/fnc_handleDamage.sqf

```diff
@@ -66,17 +66,19 @@ if (_simulationType == "tank") exitWith {
 
     // ammo was hit, high chance for cook-off
     if (_hitpoint == _ammoLocationHitpoint) then {
-        if (_damage > 0.5 && {random 1 < 0.7}) then {
+        // get cookoff probability for vehicle (default to global setting if not found)
+        private _probability = [_vehicle call CBA_fnc_getObjectConfig >> QGVAR(probability), "number", GVAR(probability)] call CBA_fnc_getConfigEntry;
```

Comment:
I think the ace setting will always be ignored because it will just use the vehicle's config first

Reply:
Maybe could use the setting as a multiplier instead? So setting the setting higher results in more common cook-off etc.